motor_blog/text/slugify.py

```diff
@@ -1,6 +1,9 @@
+import sys
 import re
 from unicodedata import normalize
 
+reload(sys)
+sys.setdefaultencoding('utf-8');
```

Comment:
This seems like a huge change. If you properly call decode("utf-8") everywhere, is it really necessary to change the default encoding?